index.src.html

```diff
@@ -22,6 +22,9 @@
     text: location; url: concept-css-style-sheet-location
     text: owner node; for: CSSStyleDeclaration; url: cssstyledeclaration-owner-node
     text: owner node; for: CSSStyleSheet; url: concept-css-style-sheet-owner-node
+spec: Fetch; urlPrefix: https://fetch.spec.whatwg.org/
+  type: dfn
+    text: ABNF; url: abnf
```

Comment:
Either Fetch or HTTP seems like the right place since you're effectively building on those anyway. INFRA is for when there's nothing else to build on.